samples/playwrightscript/testPlan.fx.yaml

```diff
@@ -0,0 +1,29 @@
+testSuite:
+  testSuiteName: testPlan Template
+  testSuiteDescription: Playwright csx example
+  persona: User1
+  appLogicalName: new_buttonclicker_0a877
+  onTestSuiteComplete: Screenshot("playwrightaction_onTestSuiteComplete.png");
+
+  testCases:
+    - testCaseName: Run Script
+      testCaseDescription: CSX example
+      testSteps: |
+        = TestEngine.Pause();
+          TestEngine.PlaywrightScript("sample.csx");
```

Comment:
This is not experimental since its not marked as such in the module as well

Reply:
Done